File: src/main/java/aurocosh/divinefavor/common/item/talismans/arrow/base/ArrowOptions.java
package aurocosh.divinefavor.common.item.talismans.arrow.base;

import java.util.EnumSet;

public enum ArrowOptions {
    BreakOnHit,
    RequiresTarget;

    public static final EnumSet<ArrowOptions> NORMAL = EnumSet.noneOf(ArrowOptions.class);
    public static final EnumSet<ArrowOptions> ARROW_BREAKS = EnumSet.of(ArrowOptions.BreakOnHit);
    public static final EnumSet<ArrowOptions> REQUIRES_TARGET = EnumSet.of(ArrowOptions.RequiresTarget);
}

File: src/main/kotlin/aurocosh/divinefavor/common/item/talismans/arrow/base/ArrowOptions.kt
package aurocosh.divinefavor.common.item.talismans.arrow.base

import java.util.*

enum class ArrowOptions {
    BreakOnHit,
    RequiresTarget;

    companion object {

        val NORMAL = EnumSet.noneOf(ArrowOptions::class.java)
        val ARROW_BREAKS = EnumSet.of(BreakOnHit)
        val REQUIRES_TARGET = EnumSet.of(RequiresTarget)
    }
}
